src-bindings/vscode_languageclient/vscode_languageclient.ml

```diff
@@ -207,5 +259,8 @@ module LanguageClient = struct
       -> Jsonoo.t Promise.t
       [@@js.call]
 
-    val registerFeature : t -> feature:StaticFeature.t -> unit [@@js.call]]
+    val registerFeature : t -> feature:StaticFeature.t -> unit [@@js.call]
+
+    val sendNotification : t -> string -> DidChangeConfiguration.t -> unit
```

Comment:
here this is partially incorrect, as the payload will not be of type `DidChangeConfiguration.t` if the notification is not for `workspace/didChangeConfiguration`, but I don't know how to encode it

Reply:
What about just making it a generic json payload and then encoding it at the call site?

Reply:
I went the generic way with a `Ojs.t` payload